docs/encrypted-innodb-tablespace-backups.md

```diff
@@ -1,432 +1,391 @@
+
+## Workflow phases
+
+The backup and restore process for encrypted InnoDB tablespaces has three distinct phases, each with different keyring-config requirements:
+
+| Phase | Description | Keyring-config requirement |
+|-------|-------------|----------------------------|
+| Backup | Run `xtrabackup --backup …` | None – plain backup works without any keyring options. |
+| Prepare | Run `xtrabackup --prepare …` (or `--apply-log`) | Required – you must supply `--component-keyring-config=<file>` matching the keyring component that the MySQL server uses for encryption. |
+| Restore | Run `xtrabackup --copy-back …` or `xtrabackup --move-back …` (followed by `--prepare` if using `--move-back`) | Required – you must provide the same keyring-config file. |
 
 !!! note
 
-    Enable only one keyring component simultaneously for each server instance. Enabling multiple keyring components is not supported and may result in data loss.
+    If you rotated the keyring using `ALTER INSTANCE ROTATE INNODB MASTER KEY`, you must use the keyring that was active when you created the backup to prepare and restore the backup.
 
-## Use the keyring vault component
+### Use the keyring vault component
 
-The `keyring_vault` component extends the server capabilities. The server uses a manifest to load the component and the component has its own configuration file.
+The MySQL server uses the `keyring_vault` component for encryption. XtraBackup uses the component's configuration file during prepare and restore operations. For information on configuring the keyring vault component on the MySQL server, see [Use the keyring vault component].
 
-The following example is a global manifest file that does not use local manifests:
+#### Phase 1: Backup
 
-```json
-{
- "read_local_manifest": false,
- "components": "file://component_keyring_vault"
-}
+No keyring configuration is needed:
+
+```bash
+xtrabackup --backup --target-dir=/data/backup --user=root
 ```
 
-The following example of a global manifest file that points to a local manifest file:
+#### Phase 2: Prepare
+
+To prepare the backup, you must provide access to the keyring. The xtrabackup binary does not communicate with the MySQL server or read the default `my.cnf` configuration file. Specify the keyring configuration file using `--component-keyring-config`.
 
-```json
-{
- "read_local_manifest": true
-}
+```bash
+xtrabackup --prepare --target-dir=/data/backup --component-keyring-config=/etc/vault.cnf
 ```
 
-The following example of a local manifest file:
+#### Phase 3: Restore
+
+After you prepare the backup, restore the backup using `--copy-back` or `--move-back`. If you use `--move-back`, run `--prepare` again after moving the files. In both cases, provide the same keyring configuration file.
 
-```json
-{
- "components": "file://component_keyring_vault"
-}
+```bash
+xtrabackup --copy-back --target-dir=/data/backup --datadir=/data/mysql \
```

Comment:
Same here:
/etc/vault.cnf is wrong. Remember we just said, component_keyring_vault.cnf should be the name